contracts/smart-account/modules/SessionKeyManagerModule.sol

```diff
@@ -115,6 +115,7 @@ contract SessionKeyManager is BaseAuthorizationModule {
         bytes32 _dataHash,
         bytes memory _signature
     ) public view override returns (bytes4) {
+        (_dataHash, _signature);
         return 0xffffffff; // do not support it here
     }
```

Comment:
What is the purpose of this function?

Reply:
imo it's just implementing ISignatureValidator interface. cc @filmakarov